Solutions/ESETPROTECT/Analytic Rules/ESETWebsiteBlocked.yaml

```diff
@@ -0,0 +1,39 @@
+id: 7b84fc5b-9ffb-4e9b-945b-5d480e330b3f
+name: Website blocked by ESET
+description: |
+  'Create alert on websites blocked by ESET.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ESETPROTECT
+    dataTypes:
+      - Syslog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Exfiltration
+  - CommandAndControl
+  - InitialAccess
+query: |
```

Comment:
Relevant Technique field is missing, can you please update the same and [refer this link ](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide)for details.